### ISSUE TITLE
Enhancement to Copy-DbaLogin

### ISSUE THREAD

**Issue by Darren150181** [dataplat/dbatools#9179]
### Summarize Functionality

At present, the process does not take into account any destination server permissions when recreating the sql login using the -Force parameter.  This could leave to a situation whereby permissions are removed on accounts supporting processes that are used to support multiple concurrent databases / processes. 

### Is there a command that is similiar or close to what you are looking for?

Yes

### Technical Details

Sort of maps to the functionality provided by Get-DbaLogin?

**Comment by wsmelton** [comment]
The copy commands are meant to be simple (as we can make them) tools for copying configurations **as-is** from a source to a destination including overwriting. The intent is keeping a destination in sync with that source. Accounting for additional configuration that exist on that destination object is out of scope in my view.

**Comment by mattcargile** [comment]
Yeah sounds like a new _cmdlet_ like `Compare-DbaLogin` or the like.

**Comment by andreasjordan** [comment]
@claude please give me some advice on this.

**Comment by andreasjordan** [comment]
@claude please implement your recommendation.
